examples/Example_header_cpp_mpl.md

```diff
@@ -0,0 +1,16 @@
+# Example snippet for a C/C++ file header licensed under a MPL-2.0 licence
+This file provides example code to be put in a C/C++ source or header file that is licensed under a MPL-2.0 licence.
+
+```
+/****************************************************************************
+**
+** Copyright (C) 2017 Pelagicore AB
+**
+** This Source Code Form is subject to the terms of the Mozilla Public
+** License, v. 2.0. If a copy of the MPL was not distributed with this
+** file, You can obtain one at http://mozilla.org/MPL/2.0/.
+** 
+** SPDX-License-Identifier: MPL-2.0
+**
+****************************************************************************/
+```
```

Comment:
I think it's a bit chunky. How about:

/*
 * blabla
 * blablabla
 */

Which would also be the same as already used in e.g. SoftwareContainer